### PR TITLE
Move HTTP API into build section

### PR DESCRIPTION
**Pull request by michaelsena**

### CODE REVIEW

File: docs/build/http/api.md [hunk 1]
```diff
@@ -1,14 +1,15 @@
...
+- You have a special use case where you directly want to make manual HTTP requests
 - You want to implement an HTTP client in a new language
 
-!!! info "Gateway mode"
-    Not all of the API methods are available if the Ceramic node runs in
-    *gateway mode*. This option disables writes, which is useful when exposing your node to
-    the internet. **Methods disabled in gateway mode will be clearly marked.**
+!!! warning "Gateway mode"
+    
+    Not all of the HTTP API methods are available if the Ceramic node runs in *gateway mode*. This option disables writes, which is useful when exposing your node to the internet. **Methods disabled in gateway mode will be clearly marked.**
```
Comment (oed) [inline docs/build/http/api.md:12]: Maybe mention gateway.ceramic.network here as an example of a node that runs in gateway mode only?

File: docs/build/http/api.md [hunk 2]
```diff
@@ -70,7 +71,7 @@ Load the state of a stream given its StreamID.
 
 **:octicons-alert-16: Disabled in gateway mode**
 
-Create a new stream, or load a stream from its genesis content. The genesis content may be signed (a DagJWS for streams of type TileDocument), or unsigned in some cases.
+Create a new stream, or load a stream from its genesis content. The genesis content may be signed (i.e. DagJWS for the [TileDocument StreamType](../../streamtypes/tile-document/overview.md)), or unsigned in some cases.
```
Comment (oed) [inline docs/build/http/api.md:74]:
```suggestion
Create a new stream, or load a stream from its genesis content. The genesis content may be signed (e.g. DagJWS for the [TileDocument StreamType](../../streamtypes/tile-document/overview.md)), or unsigned in some cases.
```

**Review by oed — COMMENTED** [review]
made some suggestions

File: docs/build/http/api.md (hunk 1 above)
Comment (michaelsena) [reply docs/build/http/api.md]: Just added a link to the tools/hosted-nodes/community-nodes.md#gateways

**Comment by michaelsena** [comment]
Thanks @oed - lgtm

**Pull request merged**